src/Card/Card.jsx

```diff
@@ -82,7 +82,10 @@ export default class Card extends Component {
 
 Card.propTypes = {
   configId: PropTypes.string.isRequired,
-  title: PropTypes.string,
+  title: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.instanceOf(Object),
```

Comment:
we should be explicit here ... instead of instanceOf(Object), this should be PropTypes.instanceOf(PlaceholderComponent)